tests/unit/test_column_noise.py

```diff
@@ -555,6 +558,51 @@ def test_write_wrong_digits(dummy_dataset):
             assert (noised_data[ssn].str[i].str.isdigit()).all()
 
 
+def test_write_wrong_digits(dummy_dataset):
+    # This is a quicker (less robust) version of the test above to run on push and pull
+    # requests. It only checks that numeric characters are noised at the correct level as
+    # a sanity check our noise is of the right magnitude
+    config = get_configuration()
+    config.update(
+        {
+            DATASETS.census.name: {
+                Keys.COLUMN_NOISE: {
+                    "street_number": {
+                        NOISE_TYPES.write_wrong_digits.name: {
+                            Keys.CELL_PROBABILITY: 0.4,
+                            Keys.TOKEN_PROBABILITY: 0.5,
+                        },
+                    },
+                },
+            },
+        }
+    )
+    config = config[DATASETS.census.name][Keys.COLUMN_NOISE]["street_number"][
+        NOISE_TYPES.write_wrong_digits.name
+    ]
+    expected_cell_noise = config[Keys.CELL_PROBABILITY]
+
+    data = dummy_dataset[["street_number"]]
+    # Note: I changed this column from string_series to street number. It has several string formats
+    # containing both numeric and alphabetically string characters.
+    noised_data = NOISE_TYPES.write_wrong_digits(
+        data, config, RANDOMNESS0, "dataset", "street_number"
+    )
+
+    # Validate we do not change any missing data
+    data = data.squeeze()
+    missing_mask = data == ""
+    assert (noised_data[missing_mask] == "").all()
+
+    # Check expected noise level
+    check_original = data[~missing_mask]
+    check_noised = noised_data[~missing_mask]
+    actual_noise = (check_original != check_noised).mean()
+    # We are setting lower and upper bounds for testing
+    assert actual_noise < expected_cell_noise
+    assert actual_noise > expected_cell_noise / 10
```

Comment:
This is just testing the magnitude of our noise accuracy and is more in line with OCR and phonetic error functions. THe more robust test will still run on schedule tests and deploy.

Reply:
@albrja Where do you define --runslow for deploy?

Reply:
Its in deploy.yml - it was in a previous PR and not this one.